### DISCUSSION TITLE
使用mingw还是检查Microsoft Visual Studio

### DISCUSSION THREAD

**Discussion by 233lol** [xmake-io/xmake#2895] [Ideas]
### Xmake 版本

2.7.1

### 操作系统版本和架构

Windows 10

### 描述问题

{
    __toolchains_mingw_x86_64 = {
        "envs",
        "mingw",
        "yasm",
        "nasm",
        "fasm",
        "go"
    },
    arch = "x86_64",
    buildir = "build",
    ccache = true,
    host = "windows",
    kind = "static",
    mingw = "C:/Users/user/scoop/apps/gcc-llvm-ucrt/current",
    mode = "release",
    ndk_stdcxx = true,
    network = "public",
    plat = "mingw",
    proxy = "http://127.0.0.1:8889",
    proxy_pac = "pac.lua",
    sdk = "C:/Users/user/scoop/apps/gcc-llvm-ucrt/current",
    theme = "default"
}

checking for mingw directory ... C:/Users/user/scoop/apps/gcc-llvm-ucrt/current
checking for Microsoft Visual Studio (x86_64) version ... no

### 期待的结果

使用mingw工具链时不检查Microsoft Visual Studio

### 工程配置

add_rules("mode.debug", "mode.release")
add_requires("fmt","vcpkg::fftw3")
target("main")
    set_kind("binary")
    add_files("src/*.cpp")
    add_packages("fmt","vcpkg::fftw3")

**Comment by waruqi** [comment]
xmake f -p mingw

**Reply by 233lol** [reply]
还是有checking for Microsoft Visual Studio (x86_64) version ... no

**Comment by CarlDegio** [comment]
确实，貌似是Windows平台上的问题
